cypress/package.json

```diff
@@ -5,12 +5,9 @@
     "node": "^16.13 || >=18"
   },
   "scripts": {
-    "copy-standard": "cp ./cypress/e2e/saucedemo.standard.ts ./cypress/e2e/saucedemo.cy.ts",
-    "copy-locked": "cp ./cypress/e2e/saucedemo.locked.ts ./cypress/e2e/saucedemo.cy.ts",
-    "copy-ignored": "cp ./cypress/e2e/saucedemo.ignored.ts ./cypress/e2e/saucedemo.cy.ts",
-    "cypress": "npm run copy-standard && cypress run",
-    "cypress-modified": "npm run copy-locked && cypress run",
-    "cypress-ignored": "npm run copy-ignored && cypress run"
+    "copy-visual": "cp ./cypress/e2e/saucedemo.visual.ts ./cypress/e2e/saucedemo.cy.ts",
```

Comment:
I think this step is not required anymore.

Since there is only one spec we can remove it and just rely on `saucedemo.cy.ts`